Ruby/Ruby.sublime-syntax

```diff
@@ -68,7 +68,7 @@ file_extensions:
   - thor
   - Thorfile
   - Vagrantfile
-first_line_match: ^#!/.*\bruby
+first_line_match: ^#!\s*/.*\bruby\b
```

Comment:
I don't think I ever saw a binary name other than `ruby` in the few years I spent working in Ruby land. Instead, all of the versioning seems to be handled by shell script manipulating the `PATH` and similar tricks.